source/_integrations/orangepi_gpio.markdown

```diff
@@ -123,3 +127,4 @@ In the table below you can see the number of usable pins for sensors per device.
 | H6 (Lite 2, One Plus) | 3/17 |
 | H6 (PI 3) | 8/17 |
 | A64 (Win Plus) | 18/28 |
+| RK3399 (PI4, PI4B) | 21/28 |
```

Comment:
```suggestion
```
As I mentioned in the PR for OPi.GPIO, it seems like, according to the technical documents, all GPIO pins support external interrupts. However as I don't have experience with Rockchip and no one has tested the other pins, we should not add this information yet as it might not be correct.